### PR TITLE
Adding WayDroid Help commands

### PR DESCRIPTION
**Pull request by ruditimmer**

### CODE REVIEW

File: qml/Main.qml [hunk 1]
```diff
@@ -219,6 +232,37 @@ MainView {
...
+            id: header3
+            title: i18n.tr('Waydroid Help')
+        }
+        Label {
+          id: helpExplain
+          anchors.top: header3.bottom
+          anchors.topMargin: 5
+          anchors.horizontalCenter: parent.horizontalCenter
+          width: parent.width * 0.9
+          text: i18n.tr ("\nYou need to run this commands into the Terminal app. \n\n") +
```
Comment (Aarontheissueguy) [inline qml/Main.qml:250]: "You need to run these commands in the terminal app."

File: qml/Main.qml [hunk 2]
```diff
@@ -219,6 +232,37 @@ MainView {
...
+                "waydroid --details-to-stdout print details (e.g. build output) to stdout, instead of writing to the log \n\n" +
+                "waydroid -v, --verbose write even more to the logfiles (this may reduce performance) \n\n" +
+                "waydroid -q, --quiet do not output any log messages"
+          wrapMode: Text.WordWrap
+        }
+
+                  Button {
+                      text: "ok"
+                      onClicked: PopupUtils.close(dialogueHide)
+                  }
```
Comment (Aarontheissueguy) [inline qml/Main.qml:263]: What does this do?

**Comment by Aarontheissueguy** [comment]
Maybe we could apply some formatting on the to make the actual commands stand out a bit from the explanations.
https://doc.qt.io/qt-5/qml-qtquick-text.html

Example:

**You need to run these commands in the terminal app.**

`waydroid -h, --help` show this help message and exit


We can use html formatting for that or whatever you prefer

**Comment by ruditimmer** [comment]
Oh yes that's sound a good idea indeed i'm not good into design ;)

**Comment by Aarontheissueguy** [comment]
Do you want to give it a try? I could take care of it later today or tomorrow if you dont have time

**Comment by ruditimmer** [comment]
I will give it a try no problem ;)

**Comment by Aarontheissueguy** [comment]
The about page is great thank you. I decided to put some more work into the help page. I tweaked the design a little and made it scrollable. I think we could add a lot more usefull commands to the list(ex. waydroid session stop).

We have one major problem to solve though. The commands we give cant be copied, which makes the page useless for longer commands. I searched the internet a bit but was not able to find a solution. Do you have an Idea on how to allow users to select and copy the commands? 
If we dont find a solution I will ask in the forum.

**Comment by ruditimmer** [comment]
Ok i will look into it about copy and paste today. thanks!

**Pull request merged**